client/src/components/HomePage/Registration.jsx

```diff
@@ -45,10 +80,12 @@ class Registration extends Component {
 
         if ('error' in data) {
           this.setState({ [`check_${data.error}`]: `Invalid ${data.error}` })
-          this.props.history.push('/registration');
           return null;
+
         } else {
-          this.props.history.push('/');
+        window.Materialize.toast("Check your email box", 3000);
+        window.history.forward('/registration-info');
```

Comment:
I don't think it works in right way, better to use `this.props.history.push`

Reply:
Okay, maybe it's don't work here because of Route in Route, so I don't have any questions for this